docs/team/yeluriketan.md

```diff
@@ -28,20 +28,20 @@ type fast enough, gitGud can get you contact management tasks done faster than t
        - Update and standardize a validation regex for data classes
     5. Update command usage messages `Refactor - Enhancement`
        - Update and standardize the command usage messages for various commands and subcommands.
-
+<br>
 * **Testing**
     1. Wrote unit/integration tests for Game, GameFriendLink and related classes.
     2. Wrote unit/integration tests for `friend --get` and `game --get` commands.
     3. Update unit/integration tests for `model` component to reach 100% class coverage, 100% method coverage, and 99% line coverage.
-
+<br>
 * **Code contributed**: [RepoSense link](https://nus-cs2103-ay2122s1.github.io/tp-dashboard/?search=YeluriKetan&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2021-09-17&tabOpen=true&tabType=authorship&zFR=false&tabAuthor=YeluriKetan&tabRepo=AY2122S1-CS2103T-W13-4%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=&authorshipIsBinaryFileTypeChecked=false)
```

Comment:
Broken markdown, caused by the `<br>` tag.
![image](https://user-images.githubusercontent.com/50147457/140652852-f8d6e66b-aa65-4c05-9700-4d649f6ad720.png)